docs/book/src/clusterctl/overview.md

```diff
@@ -20,11 +20,15 @@ Below you can find a list of main clusterctl commands:
 * [`clusterctl move`](commands/move.md) Move Cluster API objects and all their dependencies between management clusters.
 * [`clusterctl alpha rollout`](commands/alpha-rollout.md) Manages the rollout of Cluster API resources. For example: MachineDeployments.
 
+### GitHub Personal Access Token
+
+To avoid rate limiting for the public repos set `GITHUB_TOKEN`. To generate a new one [follow this](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) documentation.
```

Comment:
```suggestion
To avoid rate limiting for the public repos set the `GITHUB_TOKEN` environment variable. To generate a token [follow this](https://docs.github.com/en/authentication/
```

Reply:
>This section mentions scope and how to export the token. Maybe of value here?

Agree about adding wording that the token only needs the `repo` scope and all other scopes are not required for clusterctl. 
When we are saying clusterctl will use a person's github token it might be good to mention the level of permissions we are requesting for.